### ISSUE TITLE
Add ability to query Bugzilla

### ISSUE THREAD

**Issue by Soulofdestiny** [okurz/openqa_irc#19]
I would like to have a functionality like i know it from some other bots. 

If some 'bsc#' is mentioned in a channel, openqa-bot should automatically reply with the title of this bug and provide a clickable link

**Comment by okurz** [comment]
good idea that the bot should support this but I wonder if we need to implement it in the pluggable module or if we can reuse functionality from an existing pluggable module or another bot?
